lib/adapters/autocomplete-adapter.ts

```diff
@@ -133,7 +133,7 @@ export default class AutocompleteAdapter {
     }
 
     const filtered = !(request.prefix === "" || (triggerChar !== "" && triggerOnly))
-    return filtered ? filter(suggestions, request.prefix, { key: "filterText" }) : suggestions
+    return filtered ? filter<any>(suggestions, request.prefix, { key: "filterText" }) : suggestions
```

Comment:
@aminya I don't know if zadeh types need to be changed, but putting `Suggestion` here makes typescript fail. Not the end of the world but I would rather not have an `any` here.